src/components/CounterValue/index.js

```diff
@@ -43,10 +44,8 @@ export class CounterValue extends PureComponent<Props> {
       getCounterValue,
       ...props
     } = this.props
-
-    const date = moment(time).format('YYYY-MM-DD')
-    const v = getCounterValue(currency, counterValueCode)(value, date)
-
+    const date = new Date(moment(time).format('YYYY-MM-DD'))
```

Comment:
why not directly passing down date from props?

Reply:
Somewhere we pass it a string (didnt investigated deeply) and it was crashing :D I admit it's non optimal